sdk/ai/azure-ai-generative/azure/ai/generative/_restclient/v2022_10_01/models/_models.py

```diff
@@ -0,0 +1,11900 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from azure.core.exceptions import HttpResponseError
+import msrest.serialization
+
+
+class WorkspaceConnectionPropertiesV2(msrest.serialization.Model):
```

Comment:
Please make sure that when you generate with latest autorest that you generate without models.
None of these legacy models can be made public anyway, and you will greatly simplify your implementation as well as reduce your package size and load times by removing them.

Reply:
Will open an item in our backlog to address this